W-18506394 sap try block (#2898)
mulesoft/docs-connectors: sap/5.9/modules/ROOT/pages/sap-connector-reference.adoc

This operation works only with JCo 3.1.x and expects valid XML content in the response.

> If a function being called must impact changes in SAP, you need an open transaction. Use a `try` block.

Reviewer: ```suggestion
Use a `try` block if the function you're calling can change data in SAP.
```

> If a function being called must impact changes in SAP, you need an open transaction. Use a `try` block.

Reviewer: ```suggestion
If the function being called must impact changes in SAP, you must add a `try` block because you need an open transaction.
```
